rollingpin/deploy.py

```diff
@@ -158,8 +161,9 @@ def signal_handler(sig, _):
                     for build_hostname, build_refs in by_buildhost.iteritems():
                         build_command = ["build"] + build_refs
                         build_host = Host.from_hostname(build_hostname)
-                        (tokens,) = yield self.process_host(
+                        (results,) = yield self.process_host(
```

Comment:
Yeah this is uber-confusing, and I wanted to name it better.  I think I'm gonna go with @spladug's namedtuple recommendation to make it more understandable.